nixosModules/hermes/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  nix-std ? import (fetchTarball "https://github.com/chessai/nix-std/archive/master.tar.gz"),
+  hermes,
+}: {
+  lib,
+  config,
+  pkgs,
+  ...
+}: let
+  hermes-path = lib.meta.getExe hermes;
+  prev = config.services.hermes;
+  cfg = prev // {chains = builtins.map (pkgs.lib.filterAttrsRecursive (_: v: v != null)) prev.chains;};
```

Comment:
Can you explain  what is happening here? I am having a hard time following.

Reply:
will add. it just deletes nulls. options render nulls, and hermes fails. 

but what i am doing is as close as possible to https://github.com/terranix/terranix/blob/fc9077ca02ab5681935dbf0ecd725c4d889b9275/core/default.nix#L32

Reply:
@JonathanLorimer do you know what type to use in Nix Module options to tell `render nothing if it is null`? 

So it may be need to send PR to nix-std to respect option too.

Reply:
oh, you understood more than me:)

Reply:
@JonathanLorimer can we handle now without nix-std fixes. problem is way deeper tbh. see my coments in nix-std